### DISCUSSION TITLE
Comment format in viebrc?

### DISCUSSION THREAD

**Discussion by erwin** [Jelmerro/Vieb#336] [Q&A] [answered]
Is it possible to write comments in the viebrc file?

I've tried all of the following, but each time I get an error: `not an editor command`

```
; semicolon comment
# pound comment
// slash comment
/* c comment */
-- lua comment
```

Is there a way to include comments in the viebrc?

**Comment by ottopiramuthu** [comment, marked as answer]
Yes, use quotation marks:

```vim
" comment
set adblocker=update
```

**Comment by erwin** [comment]
That was silly of me... 

I've moved my thought process so completely to neovim's init.lua style config that I didn't try the regular Vim config comment...

Thank you!
